src/ragapp/admin-ui/client/agent.ts

```diff
@@ -40,12 +40,12 @@ export const AgentConfigSchema = z.object({
   agent_id: z.string(),
   name: z.string(),
   role: z.string(),
-  system_prompt: z.string(),
+  backstory: z.string(),
+  goal: z.string(),
+  system_prompt: z.string().nullable(),
+  system_prompt_template: z.string().nullable(),
   tools: ToolsSchema,
-  created_at: z
-    .string()
-    .or(z.date())
-    .transform((val) => new Date(val)),
+  created_at: z.number(), // Change to number
```

Comment:
better tell what this number means
```suggestion
  created_at: z.number(), // Change to number
```